packages/grid/x-data-grid-premium/src/hooks/features/aggregation/gridAggregationInterfaces.ts

```diff
@@ -112,3 +111,8 @@ export interface GridAggregationRule {
  * - items for non-available aggregation function on the column (GridColDef.availableAggregationFunctions)
  */
 export type GridAggregationRules = { [field: string]: GridAggregationRule };
+
+export interface AggregationFooterLabelColumn {
+  groupingCriteria?: string[];
+  field: string;
+}
```

Comment:
Why this interface was added? It's never used.

Reply:
Nice catch!